### ISSUE TITLE
Bug: `greater-than` formatted

### ISSUE THREAD

**Issue by littleylv** [nicklockwood/SwiftFormat#402]
Before:
```swift
if position.x > -x, position.x < x, position.y > -y, position.y < y {
}
```
After:
```swift
if position.x > -x, position.x<x, position.y> -y, position.y < y {
}
```
Causes: `'>' is not a postfix unary operator`

**Comment by nicklockwood** [comment]
I believe this is related to a known issue (as per the README):

> Under rare circumstances, SwiftFormat may misinterpret a generic type followed by an = sign as a pair of < and >= expressions. For example, the following case would be handled incorrectly:
>
> let foo: Dictionary<String, String>=["Hello": "World"]
To work around this, either manually add spaces around the = character to eliminate the ambiguity, or add a comment directive above that line in the file:
>
> // swiftformat:disable:next spaceAroundOperators
> let foo: Dictionary<String, String>=["Hello": "World"]

**Comment by littleylv** [comment]
Thanks.

**Comment by nicklockwood** [comment]
@littleylv on further investigation, I managed to find a solution. Fixed in 0.40.5
